README.md

```diff
@@ -85,3 +86,22 @@ domain = suffixList.getDomain('www.xn--85x722f.xn--55qx5d.cn');
 // domain = 'xn--85x722f.xn--55qx5d.cn'
 
 ```
+
+### Enable WebAssembly
+
+```js
+// Browser
+await publicSuffixList.enableWASM();
+
+// Node.js
+await publicSuffixList.enableWASM({
+    customFetch: fileURL => {
```

Comment:
node-fetch does not support file URLs. We have to give this example. It's the only way to turn on WebAssembly for now.